### PR TITLE
Page component

### PR DESCRIPTION
**Pull request by Duncid**
Here is a new component that should help us create pages that are consistent in terms of style and easier to make.
It retires PageHeader and SectionHeader.

### CODE REVIEW

File: sparkle/src/components/Page.tsx [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+  sm: "s-gap-3",
+  md: "s-gap-4",
+  lg: "s-gap-6",
+  xl: "s-gap-8",
+};
+
+Page.Horizontal = function ({
+  children,
+  sizing,
+  align = "left",
```
Comment (spolu) [inline sparkle/src/components/Page.tsx:139]: never used?

File: sparkle/src/stories/Page.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+import React from "react";
+
+import { ChatBubbleLeftRight } from "@sparkle/icons/solid";
+import { QuestionMarkCircle, ShakeHands } from "@sparkle/icons/stroke";
+
+import {
+  Avatar,
+  Button,
+  ChatBubbleBottomCenterTextIcon,
+  Chip,
```
Comment (spolu) [inline sparkle/src/stories/Page.stories.tsx:11]: not used

**Comment by spolu** [comment]
Left minor unused stuff comments.

Biggest question is adoption by the team but only experience will tell. LGTM

**Comment by Duncid** [comment]
> Left minor unused stuff comments.
> Biggest question is adoption by the team but only experience will tell. LGTM

So there are several things here:
- It's simpler than tailwind (for someone who has not spent time on tailwind)
- It avoids most of the inconsistencies (it always sets the right styles, it narrows down the spacing options…)
- It's the only way if we want to manage darkmode efficiently some day

**Comment by spolu** [comment]
Yep yep :+1:

**Pull request merged**